deploy/v1.1.0/avalanche.ts

```diff
@@ -31,6 +31,15 @@ export default async function deployToAvalanche(hardhat: HardhatRuntimeEnvironme
     aaveLendingPoolAddressesProviderRegistry
   } = await getNamedAccounts()
 
+  console.log(
+    deployer,
+    executiveTeam,
+    aUSDC,
+    defenderRelayer,
+    aaveIncentivesController,
+    aaveLendingPoolAddressesProviderRegistry
+  )
```

Comment:
Use conventions elsewhere in the code.

See here: https://github.com/pooltogether/v4-mainnet/blob/699abdfb5c0f07a6b5dddc3a4de022f92fc08d68/deploy/v1.0.1/polygon.ts#L36

Reply:
ohh this should be ✂️  was double checking an error with an empty param.